jupyter_book/commands/__init__.py

```diff
@@ -129,6 +158,21 @@ def build(path_book, path_output, config, toc, warningiserror, build):
                 {path_pdf_output_rel}\
             """
             )
+        if builder == "pdflatex":
+            print("Finished generating latex for book...")
+            print("Converting book latex into PDF...")
+            # Convert to PDF via tex and template built Makefile and make.bat
+            if sys.platform == "win32":
+                makecmd = os.environ.get("MAKE", "make.bat")
+            else:
+                makecmd = os.environ.get("MAKE", "make")
+            try:
+                with cd(OUTPUT_PATH):
+                    subprocess.call([makecmd, "all-pdf"])
```

Comment:
I think `run` is the recommended command to use now instead of `call`

Reply:
Roger that (https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module) I will update to use run.